src/entities/join/poolJoin.ts

```diff
@@ -18,6 +19,7 @@ export class PoolJoin {
         const { customPoolJoins } = config || {};
         this.poolJoins = {
             Weighted: new WeightedJoin(),
+            PHANTOM_STABLE: new ComposableStableJoin(),
```

Comment:
Does `PHANTOM_STABLE` cover all stable pool types?

Reply:
Only Composable Stables

Reply:
Can you add a comment that explains that here, just as a reminder for future.